docs/source/checks/simple.rst

```diff
@@ -0,0 +1,175 @@
+.. _simple:
+
+Simple
+======
+
+Description
+-----------
+
+The :ref:`simple` gives you basic information about your datasets and detects the most obvious of datashifts.
+
+Some shifts result in the change of basic metrics, that are commonly used for the analysis and exploration of datasets.
+Those metrics and statistics are calculated on both datasets and compared afterwards.
+
+The check should be used in most analyses, its metrics are commonly used to describe datasets.
+It works directly on categorical and numerical data and, with additional Precalculations, also on text data.
+
+
+Example
+-------
+
+::
+
+    from Morpheus.detector import Detector
+    from Morpheus.checks.simple_check import SimpleCheck
+
+    data_set_1 = 'examples/pokedex1.csv'
+    data_set_2 = 'examples/pokedex2.csv'
+
+    detector = Detector(
+        data_set_1,
+        data_set_2,
+        delimiter=','
+    )
+
+    detector.run(SimpleCheck())
+    detector.evaluate()
+
+The code works as follows:
+
+1. First, you create a :class:`~Morpheus.detector.Detector` object to tell Morpheus
+   which data sets you want to compare.
+2. Then you start the detector with
+   :meth:`~Morpheus.detector.Detector.run` and the checks you want to run: in this case
+   :class:`~Morpheus.checks.simple_check.SimpleCheck`.
+3. Finally, you print the result with
+   :meth:`~Morpheus.detector.Detector.evaluate`
+
+Result
+++++++
+
+The :ref:`simple` produces this report:
+
+::
+
+    Numerical Columns:
+
+    Column 'payment':
+    Metric: mean with Diff: -29.4 %, threshold: 20%
+    Metric: std with Diff: +80.2 %, threshold: 50%
+
+    (...)
+
+
+    Categorical Columns
+
+    Column 'payment_option':
+    Attribute: 'debit' with Diff: +6.0 %, categorical threshold: 5%
+    Attribute: 'cash' with Diff: -6.5 %, categorical threshold: 5%
+
+    (...)
+
+
+
+Interpretation
+++++++++++++++
+
+The results show column-wise wich shifts appear in your dataset. First the reports of your numerical columns are shown,
+after that the reports of your categorical columns.
+
+Numerical
+~~~~~~~~~
+
+In the example, below the heading 'numerical columns', you see that the column with the name 'payment' is detected
+as shifted. You see below the columnname that two numerical metrics detected a shift, *mean* and *std*
```

Comment:
```suggestion
as shifted. Below the columnname, you can see that two numerical metrics detected a shift, *mean* and *std*
```

just a small fix, that makes me feel better

Reply:
changed